staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -247,6 +252,10 @@ func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) er
 	}
 
 	proxyPath := "/apis/" + apiService.Spec.Group + "/" + apiService.Spec.Version
+	suffix := ""
```

Comment:
why the empty suffix?

Reply:
Ah, a "covfefe"-style mistake. Thanks for catching this.